tests/sharingTest.js

```diff
@@ -710,19 +956,21 @@ describe('Main: Currently testing file/folder sharing,', function () {
       })
 
       it('isShared with existent but non shared file', async function () {
+        const suffix = '123'
+        const fileName = 'newFileCreated' + suffix
+
         const provider = createProvider()
-        await getCapabilitiesInteraction(provider)
-        await getCurrentUserInformationInteraction(provider)
+        await getCapabilitiesInteraction(provider, sharer, sharerPassword)
+        await getCurrentUserInformationInteraction(provider, sharer, sharerPassword)
         await putFileWithExmptyContent(provider)
-        await shareGETExistingNonSharedFile(provider)
+        await shareGETExistingNonSharedFile(provider, '', fileName)
 
         return provider.executeTest(async () => {
-          const oc = createOwncloud()
+          const oc = createOwncloud(sharer, sharerPassword)
           await oc.login()
-          const suffix = '123'
-          return oc.files.putFileContents('newFileCreated' + suffix).then(status => {
+          return oc.files.putFileContents(fileName).then(status => {
```

Comment:
we don't need to write the file here, because this is not about putFileContents but about testing the consumer side of `isShared`
but let's leave it as it is here for now